RELEASE.md

```diff
@@ -1,5 +1,13 @@
 # Project Armadillo Release Notes
 
+## 1.0.4-SNAPSHOT
+- Upgraded Exoplayer version to 2.15.1
```

Comment:
You updated this to `2.17.1` in the next commit

Reply:
nice catch